ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -828,6 +828,10 @@ def real(self, name=None):
     def cast(self, dtype):
         return paddle_frontend.cast(self, dtype)
 
+    @with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")
+    def bmm(self, y, transpose_x=False, transpose_y=False, name=None):
+        return paddle_frontend.bmm(self, y)
```

Comment:
We can add all parameters of the function as indicated in its implementation https://github.com/unifyai/ivy/blob/c01ad72e89914df435981f4d1d5100a76dcbe6e7/ivy/functional/frontends/paddle/linalg.py#L19 😊